docs/contracts.rst

```diff
@@ -467,7 +467,7 @@ The following statements are considered modifying the state:
     }
 
 .. note::
-  ``constant`` is an alias to ``view``.
+  ``constant`` is an alias to ``view``, but this is `deprecated <https://github.com/ethereum/solidity/issues/992#issuecomment-322351715>`_ and will be removed in the next breaking release.
```

Comment:
You do not have to cite github issues in the documentation. :)
Please also mention version 0.5.0 explicitly, not only relatively.